app/models/discovery_rule.rb

```diff
@@ -36,6 +36,15 @@ class DiscoveryRule < ApplicationRecord
     end
   }
 
+  def unique_priority
+    Taxonomy.enabled_taxonomies.each do |enabled_taxonomy|
+      discovery_rule_ids  =  TaxableTaxonomy.where(taxable_type: 'DiscoveryRule', taxonomy_id: send("#{enabled_taxonomy.singularize}_ids")).pluck(:taxable_id)
```

Comment:
There are some doublespaces.

Also, put `distinct` call in front of `pluck` to remove extra entries for rules assigned to multiple orgs.